### PR TITLE
Attempt some fixes for multiple releases of a joomla version

### PR DESCRIPTION
**Pull request by wilsonge**

### CODE REVIEW

File: com_languagepack/site/models/newpack.php [hunk 1]
```diff
@@ -265,14 +265,24 @@ public function save($data)
 			'version'     => $completeVersion,
 			'alias'       => str_replace('.', '-', $completeVersion),
 			'maturity'    => 'stable',
-			'description' => '<p>This is the ' . $languageName . ' Language Pack for Joomla! ' . $joomlaVersion . '</p>',
+			'description' => '<p>This is the ' . $languageName . ' Language Pack for Joomla! ' . $joomlaVersion,
```
Comment (Llewellynvdm) [inline com_languagepack/site/models/newpack.php:268]: why are we adding p tags here, why not deal with this in the view?

Comment (Llewellynvdm) [reply com_languagepack/site/models/newpack.php:268]: and now the closing p tags is being removed.

File: com_languagepack/site/models/newpack.php [hunk 2]
```diff
@@ -265,14 +265,24 @@ public function save($data)
...
 		];
 
+		// TODO: Check if this is a string or integer in the data and delete one of these.
+		if ($releaseVersion === 1 || $releaseVersion === '1')
+		{
+			$arsReleaseData['description'] .= '</p>';
+		}
+		else
+		{
+			$arsReleaseData['description'] .= ' (v' . $releaseVersion . ')</p>';
```
Comment (Llewellynvdm) [inline com_languagepack/site/models/newpack.php:280]: hmm I see, still wonder why the p tags are here.

Comment (wilsonge) [reply com_languagepack/site/models/newpack.php:280]: This is what renders in the ARS View in the HTML Interface

**Pull request merged**